client/server/middleware/unsupported-browser.js

```diff
@@ -28,15 +56,7 @@ function allowPath( path ) {
 		? path.replace( new RegExp( `^/${ prefixedLocale }` ), '' )
 		: path;
 
-	const allowedPaths = [
-		'/browsehappy',
-		'/log-in',
```

Comment:
I'd love to hear more about what data we have on which page works on which browser and which doesn't. The word "likely" in the new comment above bothers me a little bit. It sounds like we could go further to verify what works and what doesn't.

Reply:
I should probably update the comment. I tested and the pages I removed from here definitely do not work